doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+
+    Fiware-ServicePath: /Madrid/Gardens/ParqueNorte/Parterre1
+		
+Queries without Fiware-ServicePath resolve to `/#`.
+	
+It is assumed that all the entities created without Fiware-ServicePath (or that don't include service path information in the database) 
+belongs to a root scope `/` implicitely. All the queries without using Fiware-ServicePath (including subscriptions) are on `/#` implicitly. 
+
+While entities belong to services and servicepaths, subscriptions and registrations belong only to the service. 
+The servicepath in subscriptions and registrations doesn't denote sense of belonging, but is the expression of the 
+query associated to the subscription or registration.
+
+Fiware-ServicePath header is ignored in `GET /v2/subscriptions/{id}` and `GET /v2/registrations/{id}` operations, 
+as the id fully qualifies the subscription or registration to retrieve.
+
+Fiware-ServicePath header is taken into account in `GET /v2/subscriptions` and `GET /v2/registrations` in order to narrow 
+down the results to subscriptions/registrations that use exactly that service path as query.
+
+At the present moment hierarchical service paths (i.e. the ones using ending with #) are not allowed in registrations.
+Fiware-Service and Fiware-ServicePath are set at entity creation time using HTTP headers in the entity creation REST request.
+		
+    "servicePath": "test/example" 		
+
+[Top](#top)
+ 
+ 
+## 21. Fiware-Total-Count
+
+It is an optional header used in responses.
+NGSIv2 implements a pagination mechanism in order to help clients to retrieve large sets of resources. This mechanism works for all listing operations in the API (e.g. `GET /v2/entities`, `GET /v2/subscriptions`, `POST /v2/op/query`, etc.).
+
+The mechanism is based on three URI parameters:
+
+i)   limit, in order to specify the maximum number of elements (default is 20, maximum allowed is 1000).
+
+ii)  offset, in order to skip a given number of elements at the beginning (default is 0)
+
+iii) count (as option), if activated then a Fiware-Total-Count header is added to the response, with a count of total elements.
+	    
+[Top](#top)
+
+
+## 22. Fiware-Correlator
+
+Fiware-Correlator is a response HTTP header.
+Fiware-Correlator is used as HTTP header in forwarding messages, notifications and responses.
+This correlator id is either transferred from an incoming request, or, if the incoming request doesn't carry any HTTP header Fiware-Correlator, the correlator is generated by the Orion context broker and then used in the log file. 
+It can be either `N/A` or it is a string in the UUID format.
+The correlator id is a common identifier among all applications involved in the `message chain` for one specific request.	
+Fiware-Correlator is a NON-MODIFIABLE header in custom notifications.    
+Any attempt of doing so (e.g. `"httpCustom": { ... "headers": {"Fiware-Correlator": "foo"} ...}` will be ignored.	
```

Comment:
This text has been copied from some existing documentation? (it seems so, but in this case I'm not so sure...).

Reply:
Yes, text has been copied but not from single file.

Reply:
Fixed in a427bd9